components/overflow-group/overflow-group.js

```diff
@@ -34,6 +34,7 @@ function createMenuItem(node) {
 	const childText = node.text || node.firstChild && (node.firstChild.label || node.firstChild.text || node.firstChild.textContent.trim());
 	const disabled = node.disabled;
 	return html`<d2l-menu-item 
+		@click=${node.onclick}
```

Comment:
I think you want the `d2l-menu-item-select` event, as the click event likely won't work with the keyboard. Also just passing `node.onclick` likely won't work in all scenarios like if a click event handler was added in JS. The [old code](https://github.com/BrightspaceUI/button-group/blob/master/d2l-button-group-responsive-behavior.js#L311) might be the best way.